laevatein/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 27
     buildToolsVersion "26.0.3"
```

Comment:
You sould update buildToolVersion too.
Or delete this line.

Reply:
fixed this commit: 7e60b06